xeth/xeth.go

```diff
@@ -1044,6 +1044,7 @@ func (self *XEth) Transact(fromStr, toStr, nonceStr, valueStr, gasStr, gasPriceS
 	if err != nil {
 		return "", err
 	}
+	self.EthereumService().TxPool().SetLocal(signed)
```

Comment:
@bas-vk done, added it to sendRawTransaction too. I think you're right about that, a raw transaction might have been signed somewhere else but if a local application intends to add it then the local client should always accept it.